test/test.js

```diff
@@ -35,7 +40,10 @@ test('equals to sample', async t => {
 
 test('check internal structure', async t => {
   // Create deck as in previous example
-  const apkg = new AnkiExport('deck-name');
+  const apkg = new AnkiExport('deck-name', {
+    template,
+    DbClass: sql.Database
```

Comment:
Should be `sql: sql.Database`. 

The reason why this doesn't break tests is you importing `AnkiExport` from `index.js` instead of `exporter.js` https://github.com/ewnd9/anki-apkg-export/pull/8/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R6